lib/context.js

```diff
@@ -44,25 +44,31 @@ Context.prototype.timeout = function (ms) {
 };
 
 /**
- * Set test timeout `enabled`.
+ * Set or get test timeout `enabled`.
  *
  * @api private
  * @param {boolean} enabled
  * @return {Context} self
  */
 Context.prototype.enableTimeouts = function (enabled) {
+  if (!arguments.length) {
```

Comment:
(fwiw, this is a breaking change for those who are consuming it directly.  as it's undocumented, they shouldn't be.)

Reply:
So should i remove the arguments check from enableTimeouts?

Reply:
sorry to confuse; no, please leave it.  I don't have any evidence this is used directly.

Reply:
(looks like you'll need to revert)